src/App.tsx

```diff
@@ -1,20 +1,16 @@
-import reactLogo from '@/assets/react.svg';
-
-import { HeaderComponent } from './components/header/Header.component.tsx';
+import AboutComponent from '@/components/about/About.component.tsx';
+import FooterComponent from '@/components/footer/Footer.component.tsx';
+import HeaderComponent from '@/components/header/Header.component.tsx';
 
 import './App.css';
 
 function App() {
     return (
-        <>
+        <div className="about">
             <HeaderComponent />
```

Comment:
![Screenshot 2024-04-29 at 22 13 12](https://github.com/lesiaturuta/fe-react-2024/assets/115080771/6a28b6c6-6fd1-4890-9b0e-3d9133a8dd23)

design has all blocks in one line. Need to fix.  And image on mobile has border-radius.

Reply:
fixed img.